dp-cluster-setup-utility.py

```diff
@@ -1321,18 +1391,27 @@ class DpProxyTopology:
            <enabled>true</enabled>
         </provider>
      </gateway>
-     <service>
-        <role>AMBARI</role>
-        <version>0.2.2.0</version>
-        <url>{ambari_protocol}://{ambari_host}:{ambari_port}</url>
-     </service>
+     {cluster_manager}
      {ranger}
      {atlas_api}
      {dpprofiler}
      {beacon}
      {streamsmsgmgr}
   </topology>"""
 
+  def get_template(slef, knox, topology_util):
```

Comment:
I believe this should be 'self' and not 'slef'

Reply:
we change